py/dml/types.py

```diff
@@ -88,6 +89,32 @@ def __str__(self):
 # anonymous structs used by global typedefs, label -> type
 global_anonymous_structs = {}
 
+def check_named_types(t):
+    '''Checks that a type does not reference a non-existing type'''
+    if isinstance(t, TNamed):
```

Comment:
match/case?

Reply:
I rather not piss off fisketur.
Tangentially, the very existence of `check_named_types` is questionable considering `_ = safe_realtype(t)`, from what I can tell, fits exactly the same niche. I employed it for verifying shared method signatures because it's what we employ to verify _regular_ method signatures. I'm not sure if there's anything that makes `check_named_types` preferable. What's your opinion?